### DISCUSSION TITLE
roadmap to v4

### DISCUSSION THREAD

**Discussion by michabbb** [filamentphp/filament#13946] [General]
hi all,
i wasn´t able to find any official roadmap. is there one?
i would like to know when v4 is planned to be released or at least what the desired timeframe looks like ;)

thanks for any info!

**Comment by leandrocfe** [comment]
https://github.com/orgs/filamentphp/projects/2/views/20
https://github.com/filamentphp/filament/issues?q=is:open+is:issue+milestone:v4
